src/braket/ahs/driving_field.py

```diff
@@ -83,6 +83,30 @@ def detuning(self) -> Field:
         r"""Field: global detuning (:math:`\Delta(t)`). Time is in s, and value is in rad/s."""
         return self._detuning
 
+    def stitch(self, other: DrivingField, boundary: str = "mean") -> DrivingField:
```

Comment:
Should we make boundary an Enum instead?

Reply:
updated, see below (TimeSeries)